tests/runtime/http_server/test_http_server.py

```diff
@@ -43,6 +43,15 @@
     runtime_http_test_server,
 )
 
+## Fixtures #####################################################################
+
+
+@pytest.fixture(scope="session")
```

Comment:
Why is this session scoped? I'm wondering if this is hiding the real issue of concurrent requests with multiple connections?

Reply:
Yeah without the session scope it's basically the same thing we had before, and it fails on the second stream test

Reply:
Hm, ok, that actually seems like this doesn't really solve the problem then, right? Is this a property of the `TestClient` itself or rather a property of the server app being connected to? If it's the former, this is fine as the "problem" lies in the testing infrastructure, but if it's the later, this is just hiding that and is probably not the right fix.

Reply:
Removing session scoping, I agree it's not the right thing to do in this case.